presto-native-execution/presto_cpp/main/types/PrestoToVeloxConnector.cpp

```diff
@@ -13,6 +13,10 @@
  */
 
 #include "presto_cpp/main/types/PrestoToVeloxConnector.h"
+#include "presto_cpp/presto_protocol/connector/hive/HiveConnectorProtocol.h"
```

Comment:
Ideally it would be great to split the `PrestoToVeloxConnector` files as well. Can be done as a follow up.

Reply:
Yes, I was planning to have a followup PR for this